lib/rails_admin.rb

```diff
@@ -33,6 +34,28 @@ def self.config(entity = nil, &block)
       RailsAdmin::Config
     end
   end
+
+  # Backwards-compatible with safe_yaml/load when SafeYAML isn't available.
+  # Evaluates available YAML loaders at boot and creates appropriate method,
+  # so no conditionals are required at runtime.
```

Comment:
two conditions for defining `RailsAdmin.yaml_load` depending on available gems. But the nicest part is, that this method can be customized by any end user :)